tests/mol-embed/EPUB/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-02-18</dc:date>
+    <dc:description>Tests whether the Reading System implements the control of embedded videos via media overlay.</dc:description>
+    <dc:identifier id="pub-id">mol-embed</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Media overlay with embedded video</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="belongs-to-collection">should</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#sec-embedded-media</meta>
```

Comment:
I think this needs to be more specific — many normative statements in this section and it's not clear which are under test (and, correspondingly, which aren't tested here and need to be tested later).

Reply:
Yep. The tests were to be checked first before I added the right anchors. 

Interestingly, this basic tests covers quite a lot of ground in the section, though.